//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// abc_T_A.h
//
// Code generation for function 'abc_T_A'
//

#ifndef ABC_T_A_H
#define ABC_T_A_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void abc_T_A(double A_init, double A_wayp, double A_max, double J_max,
             double J_min, double T, double t[7]);

#endif
// End of code generation (abc_T_A.h)
